IOPool/Input/src/RootInputFileSequence.cc

```diff
@@ -559,7 +559,8 @@ namespace edm {
       findFileForSpecifiedID_.reset(new std::unordered_multimap<size_t, size_t>);
       auto hasher = std::hash<std::string>();
       for(auto fileIter = fileIterBegin_; fileIter != fileIterEnd_; ++fileIter) {
-        findFileForSpecifiedID_->insert(std::make_pair(hasher(fileIter->logicalFileName()), fileIter - fileIterBegin_));
+	 //        findFileForSpecifiedID_->insert(std::make_pair(hasher(fileIter->logicalFileName()), fileIter - fileIterBegin_));
+         findFileForSpecifiedID_->insert(std::make_pair((hasher(fileIter->logicalFileName()) & 0xffffffff), fileIter - fileIterBegin_));
```

Comment:
To be consistent, the mask should also be applied to the place where one looks up data in findFileForSpecifiedID_